### ISSUE TITLE
unexpected 'class' (T_CLASS), expecting identifier 

### ISSUE THREAD

**Issue by Moosjes** [Sibyx/phpGPX#24]
# Problem description

I get the following error:
Parse error: syntax error, unexpected 'class' (T_CLASS), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in vendor/sibyx/phpgpx/src/phpGPX/Models/Track.php on line 51

# Code snippet you use

```php
<?php
use phpGPX\phpGPX;

$gpx = new phpGPX();

$file = $gpx->load('scripts/example.gpx');

foreach ($file->tracks as $track)
{
    // Statistics for whole track
    $track->stats->toArray();

    foreach ($track->segments as $segment) {
        // Statistics for segment of track
        $segment->stats->toArray();
    }
}
```

**Comment by Sibyx** [comment]
Hi,
can you please provide me yours `.gpx` file so I can reproduce the problem? Also, can you please tell me which version of the library are you currently using?

**Comment by Moosjes** [comment]
Well it does not seem to matter what GPX file I use but this is one for example
[example.zip](https://github.com/Sibyx/phpGPX/files/1702389/example.zip)

In src/phpGPX/Models/Track.php on line 51 there is a piece of code:
`usort($points, array(DateTimeHelper::class, 'comparePointsByTimestamp'));`

If I remove the ::class from this line it works. I have no clue to what kind of problems this creates in the rest of the code but with this change I can go on with my code.

It is the latest version

**Comment by Sibyx** [comment]
I know that problem don't have to be connected to your GPX file. I just want to collect all relevant data before deeper investigation. 

Which version of PHP do you use?

**Comment by Moosjes** [comment]
PHP Version 5.4.16

**Comment by Sibyx** [comment]
Ah,
I see the problem. The class name resolution using `::class` have been introduced in PHP 5.5. We have wrong PHP support in `composer.json`. I will check the codebase of the project and if it will be possible to support PHP 5.4 I will do necessary changes if not, the PHP version support will be changed to PHP 5.5.

Anyway, I recommend you to migrate to PHP 7 if it's possible

**Comment by Moosjes** [comment]
That is something I am working on but it's a little bit more work than I wanted :P

**Comment by Sibyx** [comment]
I have probably fixed support for PHP 5.4, but I am unable to run tests because we are using PHPUnit 6.5.

So please let me know if you will have some problems with the library with PHP 5.4
